modules/combined/tests/contact_verification/hertz_cyl/half_symm_q4/tests

```diff
@@ -149,7 +148,7 @@
     max_parallel = 1
     heavy = true
     superlu = true
-    skip = 'Issue 6528'
+    skip = 'Waiting for additional code checkin to fix issue causing test failure.'
```

Comment:
I can merge this as is if you want, but generally the real estate available for printing descriptive messages is very limited in the TestHarness printout. That's why we like short messages or just issue numbers.